### PR TITLE
Added Lmax option and fixed broken legend

### PR DESCRIPTION
**Pull request by adam-broussard**
You can now use "--Lmax=100000" for example as an option to set Lmax

### CODE REVIEW

File: hsc/hsc_consistency.py [hunk 1]
```diff
@@ -9,17 +9,35 @@
 from copy import deepcopy
 
 def main():
-    if len(sys.argv)<3:
+
+    fnames = []
+    Lmax = -1
+    savefig = False
+    figsize = (7,7)
+    for argnum in range(1, len(sys.argv)):
```
Comment (slosar) [inline hsc/hsc_consistency.py]: Come on, Adam, just implement this right: 
https://docs.python.org/3/howto/argparse.html

File: hsc/hsc_consistency.py [hunk 2]
```diff
@@ -61,20 +79,31 @@ def main():
...
+    fig, splist = plt.subplots(Ntomo,Ntomo, figsize = figsize)
     clrcy='rgbycmk'
-    for i,s in enumerate(saccs):
-        s.plot_vector(out_name=None,clr=clrcy[i],lofsf=1.01**i,
-                      label=s.tracers[0].exp_sample)
-    els=saccs[0].binning.binar['ls']
-    plt.plot(els,mean,'k-',lw=2)
+    splist = np.array(splist).T.tolist()
+    for (i,s) in enumerate(saccsin):
+        for x in range(len(splist)):
```
Comment (slosar) [inline hsc/hsc_consistency.py]:
```suggestion
        for x,sx  in enumerate(splist):
```

File: hsc/hsc_consistency.py [hunk 3]
```diff
@@ -61,20 +79,31 @@ def main():
...
+            for y in range(x, len(splist[x])):
```
Comment (slosar) [inline hsc/hsc_consistency.py]:
```suggestion
            for y,sxy in enumerate(sx):
```

File: hsc/hsc_consistency.py [hunk 4]
```diff
@@ -61,20 +79,31 @@ def main():
...
+                s.plot_vector(splist[x][y], plot_corr = [[x, y]], clr=clrcy[i],lofsf=1.01**i,
```
Comment (slosar) [inline hsc/hsc_consistency.py]:
```suggestion
                s.plot_vector(sxy, plot_corr = [[x, y]], clr=clrcy[i],lofsf=1.01**i,
```

Comment (slosar) [reply hsc/hsc_consistency.py]: etc... now you have sxy.

File: hsc/hsc_consistency.py (hunk 1 above)
Comment (adam-broussard) [reply hsc/hsc_consistency.py]: My apologies - I am not very familiar with running python scripts in terminal.  I have now made these changes.

File: hsc/hsc_consistency.py (hunk 4 above)
Comment (adam-broussard) [reply hsc/hsc_consistency.py]: Thank you for the suggestion!

**Comment by adam-broussard** [comment]
This now also depends on my request at LSSTDESC/SACC#13, so that should likely be approved before merging this pull request.

**Pull request merged**